projects/packages/my-jetpack/_inc/context/notices/types.ts

```diff
@@ -7,6 +7,7 @@ export type Notice = {
 		actions?: {
 			label: string;
 			onClick: () => void;
+			isLoading?: boolean;
```

Comment:
Now that we have control of the notice component, could we customize the loading state to say "Connecting...." for this action? 

Maybe a custom loading text prop?